javascript/chronometer.js

```diff
@@ -1,26 +1,59 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = 0;
   }
   startClick(callback) {
-    // ... your code goes here
-  }
+    this.intervalId = setInterval(() => {this.currentTime++}, 1000);
+  }  
   getMinutes() {
-    // ... your code goes here
+    let minutes = Math.floor(this.currentTime / 60);
+    return minutes;
   }
+
   getSeconds() {
-    // ... your code goes here
+    let seconds = Math.floor(this.currentTime % 60);
+    return seconds;
   }
+  
   twoDigitsNumber() {
-    // ... your code goes here
-  }
+    let minDec;
+    let minUni;
+    let splitMin = this.getMinutes().toString().split("");
+      if (this.getMinutes() < 10) {
+        minUni = splitMin[0]
+        minDec = "0"
+        }
+      if (this.getMinutes() >= 10) {
+        minUni = splitMin[1]
+        minDec = splitMin[0]
+        }
+
+    let secDec;
+    let secUni;
+    let splitSec = this.getSeconds().toString().split("");
+      if (this.getSeconds() < 10) {
+        secUni = splitSec[0]
+        secDec = "0"
+      }
+      if (this.getSeconds() >= 10 ) {
+        secUni = splitSec[1];
+        secDec = splitSec[0]
+        
+      } 
+
+    return `${minDec}${minUni}:${secDec}${secUni}`
+  }
+
   stopClick() {
-    // ... your code goes here
+    clearInterval(this.intervalId)
   }
+
   resetClick() {
-    // ... your code goes here
+    this.currentTime = 0;
   }
+
   splitClick() {
-    // ... your code goes here
+    return this.twoDigitsNumber() 
```

Comment:
`return `${this.twoDigitsNumber(this.getMinutes())}:${this.twoDigitsNumber(this.getSeconds())}`;`